src/main/java/jenkins/plugins/http_request/HttpRequestExecution.java

```diff
@@ -516,7 +516,7 @@ public void checkServerTrusted(X509Certificate[] chain, String authType) {
 
 		@Override
 		public X509Certificate[] getAcceptedIssuers() {
-			return null;
+			return new X509Certificate[0];
```

Comment:
This change makes the implementation comply with the [javadoc comment](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/X509TrustManager.html#getAcceptedIssuers--) of the upstream class.  It says:

```
Returns:
a non-null (possibly empty) array of acceptable CA issuer certificates.
```